18F/C2
# [Delivers #97242150] disallow assigning requester as approver and vice-versa

Story: https://www.pivotaltracker.com/story/show/97242150

Testing:

* create new NCR work order
* verify you are not listed amongst the possible approvers
